app/views/proposal/index.html.haml

```diff
@@ -97,5 +97,5 @@
                   class: 'btn btn-default', id: "edit_proposal_#{event.id}"
   .row
     .col-md-12
-      - if @conference.cfp_open? || (current_user.has_role? :organizer, @conference)
+      - if @conference.cfp_open?
```

Comment:
With that condition, if you have role of organizer for conference, you can still see "New Proposal" button if call_for_papers period is closed. Also, you can submit new proposal after call_for_papers period is closed.